java-client/scripts/download_thrift.sh

```diff
@@ -48,21 +48,4 @@ if ! $thrift -version | grep "0.11.0" ; then
     exit 1
 fi
 
-TMP_DIR=./gen-java
```

Comment:
It seems `thrift` will not be used any where in java-client, can we remove this file entirely?

Reply:
thrift will be used when `mvn compile` or `mvn test`. If i remove the file , if i remove the file  https://github.com/apache/incubator-pegasus/blob/master/.github/workflows/test_python-client.yml#L68 will be add in to github action

Reply:
OK, let's keep it in current step. We can do that in next patches.